api_auth/urls.py

```diff
@@ -1,12 +1,13 @@
 from django.urls import path
 from rest_framework.urlpatterns import format_suffix_patterns
 
-from api_auth.User.UserView import (UserLogin, UserRegistration, UserUpdate, UserGetInfo)
+from api_auth.User.UserView import (UserLogin, UserRegistration, UserUpdate, UserGetInfo, ChangePassword)
 
 urlpatterns = [
     path('Login', UserLogin.as_view()),
     path('registration', UserRegistration.as_view()),
     path('User/Update', UserUpdate.as_view()),
     path('User/Get/Info/<int:user_id>', UserGetInfo.as_view())
+    path('ChangePassword/<int:user_id>', ChangePassword.as_view())
```

Comment:
If we are not going to use the user_id parameter, there's no need to keep it on the URL and in the code. (The same for User Get Info)

Reply:
nice catch, forgot to remove